src/rocprof_compute_soc/analysis_configs/gfx906/1500_address_processing_unit_and_data_return_path_ta_td.yaml

```diff
@@ -0,0 +1,299 @@
+# AUTOGENERATED FILE. Only edit for testing purposes, not for development. Generated from utils/unified_config.yaml. Generated by utils/split_config.py
```

Comment:
"address_processing_unit_and_data_return_path_ta_td", this is not a good practice for naming conversion.
If we really want to unify those, let's keep the whole full name only, like address_processing_unit_and_data_return_path. 

And lower case L1 and L2 are bad idea that engineers complained in the past.